docs/glossary.rst

```diff
@@ -0,0 +1,20 @@
+Glossary
+========
+
+.. glossary::
+
+   dashboard
+      Main page where you can see all your projects with their build status
+      and import a new project.
+
+   profile page
+      Page where you can see the projects of a certain user.
+
+   project home
```

Comment:
I don't really understand the difference between home & page. Are you trying to specify the content? `home` is weird for that I think, and we've never really separated the content of the page vs. the page itself conceptually.

Reply:
I can make "project home" an alias to "project page" and simlpify these definitions